nb2/bot/slack_bot.py

```diff
@@ -123,6 +148,20 @@ def remember(self, target_slack_user_id: str, quote: str):
 
         return self.Result(ok=True, message="Memory stored!")
 
+    def remind(self, nostalgia_user_target: str, slack_user_targets: List[str]):
+        person = get_person_name_by_slack_user_id(nostalgia_user_target)
+
+        if person is None:
+            return self.Result(ok=True, message="I don't remember that person.")
```

Comment:
I think NB1 reiterates the `nostalgia_user_target` right?

Maybe we can have the message as `f"I don't remember {nostalgia_user_target}"`?

Reply:
`nostalgia_user_target` I think is just the jiberish slack user id, but maybe I could use the `fetch_user_info` method to get the user tag or first name